v2/jwt/error_test.go

```diff
@@ -0,0 +1,35 @@
+package jwt
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDoNotLooseUnderlyingError(t *testing.T) {
+	internalError := fmt.Errorf("my nice error")
+	err := errNotValidNow(internalError)
+
+	require.ErrorIs(t, err, internalError)
+}
+
+func TestIsNotValidNowError(t *testing.T) {
+	err := errNotValidNow(errors.New(""))
+
+	require.True(t, errors.Is(err, ErrNotValidNow))
+}
+
+func TestRandomErrorIsNotNotValidNowErr(t *testing.T) {
+	err := errors.New("random Error")
+	require.False(t, errors.Is(err, errExpiredType{}))
```

Comment:
Why do we use `ErrNotValidNow` in the happy test but not in the sad?

Reply:
I forgot to change it.